src/Common/JSONParsers/SimdJSONParser.h

```diff
@@ -16,6 +18,251 @@ namespace ErrorCodes
     extern const int CANNOT_ALLOCATE_MEMORY;
 }
 
+class SimdJSONStringFormatter
```

Comment:
Please add a big comment to these classes about their origin implementation is in simdjson. I found out this accidentally, but actually it is really important detail.

Furthermore, please adjust the casing of the functions to follow `camelCasing` as the rest of our code base.